mmdet3d/datasets/__init__.py

```diff
@@ -26,6 +29,10 @@
 from .utils import get_loading_pipeline
 from .waymo_dataset import WaymoDataset
 
+pwd = os.getcwd()
+sys.path.append(os.path.join(pwd, 'projects/dummy_PETR'))
+from mmdet3d_plugin.datasets.transforms import *  # noqa: E402,F401,F403
```

Comment:
These lines of code are not necessary. We can add transformers  in projects/petr

Reply:
modify according to this: https://github.com/open-mmlab/mmdetection3d/pull/2170/files#diff-693bc44c728cd63f1c0713a417217f1e56044f4bec431e4054b830a243788da8R6